docs/indepth/installation.rst

```diff
@@ -379,6 +379,58 @@ Your firewall will need to allow access to the ports your applications are expos
    Keep in mind the consequences of exposing unsecured services to the Internet.
    Both applications with exposed ports and applications accessed via links will be accessible by anyone on the Internet.
 
+ZFS Backend Configuration
+-------------------------
+
+The ZFS backend requires ZFS to be installed.
+
+
+Installing ZFS on CentOS 7
+..........................
+
+Installing ZFS requires the kernel development headers for the running kernel.
+Since CentOS doesn't provide easy access to old package versions,
+the easiest way to get appropriate headers is to upgrade the kernel and install the headers.
```

Comment:
The ZFS section appears as an addition in the diff because it has been moved, not because it has been added.  I'd rather not change the text in a moved section, because it will make resolving merge conflicts more painful.  Added FLOC-1975 to address this and next few comments in the future.